src/PinAssignment/PackagePinsLoader.cpp

```diff
@@ -20,7 +20,50 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 #include "PackagePinsLoader.h"
 
+#include <QFile>
+
 namespace FOEDAG {
 
-PackagePinsLoader::PackagePinsLoader() {}
+PackagePinsLoader::PackagePinsLoader(PackagePinsModel *model, QObject *parent)
+    : QObject(parent), m_model(model) {}
+
+PackagePinsLoader::~PackagePinsLoader() {}
+
+bool PackagePinsLoader::load(const QString &fileName) {
+  if (!m_model) return false;
+
+  QFile file{fileName};
+  if (!file.open(QFile::ReadOnly)) {
+    return false;
+  }
+
+  QString content = file.readAll();
+#if QT_VERSION >= QT_VERSION_CHECK(5, 14, 0)
```

Comment:
Why do we need this check? We already have a Qt version defined.

Reply:
Because ubuntu default version is 5.12 is not the latest. the latest is 5.15. There is difference between them.

Reply:
Add supported Qt versions in build documentation.